packages/app/src/cli/commands/app/function/run.ts

```diff
@@ -42,10 +42,43 @@ export default class FunctionRun extends Command {
       path: flags.path,
       userProvidedConfigName: flags.config,
       callback: async (_app, ourFunction) => {
+        let functionExport = '_start'
+
+        if (flags.export !== undefined) {
+          this.log('Using export ${flags.export} from the --export flag.')
```

Comment:
I wonder if these logs should be debug/verbose output, and the Function Runner itself should just report what export it's running.